snap/info.go

```diff
@@ -799,10 +799,12 @@ func ReadInfo(name string, si *SideInfo) (*Info, error) {
 	}
 
 	st, err := os.Stat(MountFile(name, si.Revision))
-	if err != nil {
+	if err != nil && !os.IsNotExist(err) {
```

Comment:
can we have a comment about when this can happen, given the snap.yaml is there?

Reply:
Sure